api/services/restaurantService.js

```diff
@@ -1,9 +1,20 @@
 import { restaurantDao } from '../models/index.js';
 
-const districts = async() => {
+const districts = async () => {
   return await restaurantDao.districts();
-}
+};
 
-export default {
-  districts
-}
\ No newline at end of file
+const getRestaurantInfo = async (restaurantId) => {
+  const existRestaurantId = await restaurantDao.existRestaurantId;
+
+  if (!!existRestaurantId) {
+    const error = new Error('INVALID_RESTAURANT_ID');
+    error.statusCode = 404;
+
+    throw error;
+  }
+
+  return await restaurantDao.getRestaurantInfo(restaurantId);
```

Comment:
exist를 따로 만들 필요는 없을 것 같습니다.

```suggestion
const getRestaurantInfo = async (restaurantId) => {
  const restaurant = await restaurantDao.getRestaurantInfo(restaurantId);

  if (!restaurant) {
    const error = new Error('INVALID_RESTAURANT_ID');
    error.statusCode = 404;

    throw error;
  }

  return restaurant
```